html-practices/my-new-change/script.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+if [ $# -eq 0 ]; then
+    echo "No target branch name provided, command ignored"
+    exit 1
+fi
```

Comment:
**suggestion** 💭
There's a better way to do this:
```suggestion
    echo "No target branch name provided, command ignored"
    exit 0
fi
```

<sub><sup>I'm using [conventional comments](https://conventionalcomments.org/) formatting to improve comments readability and reader's expectations in this PR.</sup></sub>